.github/workflows/artifacts-build.yml

```diff
@@ -237,7 +237,7 @@ jobs:
     - name: Install Dart
       uses: dart-lang/setup-dart@0a8a0fc875eb934c15d08629302413c671d3f672 # v1.6.5
       with:
-        sdk: 3.7.0
+        sdk: 3.8.0
```

Comment:
the build scripts works with 3.7.0 but not 3.8.0, which is why I haven't updated this yet.